test/js/samples/reactive-class-optimized/input.svelte

```diff
@@ -0,0 +1,31 @@
+<script context="module">
+	let nonReactiveModuleVar = Math.random();
+	let reactiveModuleVar = Math.random();
+</script>
+
+<script>
+	import { reactiveStoreVal, unreactiveExport } from './store';
+
+	nonReactiveGlobal = Math.random();
+	const reactiveConst = {x: Math.random()};
+
+	$: reactiveDeclaration = reactiveModuleVar * 2;
+
+	reactiveModuleVar += 1;
+	if (Math.random()) {
+		reactiveConst.x += 1;
+	}
+</script>
+
+<!--These should all get updaters because they have at least one reactive dependency-->
+<div class:update1={reactiveModuleVar}></div>
```

Comment:
actually `moduleVar` will never be reactive as it does not belong to any component instance

Reply:
Interesting! If it's okay with you, I would like to file an issue for that and solve it in a different PR. It's being indicated as being reactive by `is_dynamic` and I worry that changing that function to fix this will have other effects and would prefer to isolate that (and the testing for it) to a new PR.

My PR won't cause a regression even though it treats the module var as reactive. https://svelte.dev/repl/3a6d024461d74e43b5fdf7ea144a9a80?version=3.32.0 shows that Svelte already creates a `toggle_class` for a module var in the update function so my PR won't change that behavior at all.

Reply:
Filed an issue at https://github.com/sveltejs/svelte/issues/5943 and I'll hopefully have a fix out within a week